glean-core/src/metrics/time_unit.rs

```diff
@@ -0,0 +1,58 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
+
+use crate::error::{Error, ErrorKind};
+
+/// Enumeration of different resolutions supported by the
```

Comment:
Wording suggestion: Just say "Different time resolutions supported by ..." (that is: leave out `enumeration`)